kalite/distributed/static/js/distributed/distributed-server.js

```diff
@@ -35,14 +35,14 @@ function show_api_messages(messages) {
     }
 }
 
-function force_sync() {
+function force_sync(zone_id, device_id) {
```

Comment:
Identifier 'force_sync' is not in camel case.<br>Identifier 'zone_id' is not in camel case.<br>Identifier 'device_id' is not in camel case.